### PR TITLE
SRV-33 database migrations and fixtures for network hosts

### PR DESCRIPTION
**Pull request by c3zi**

### CODE REVIEW

File: app/Models/NetworkHost.php [hunk 1]
```diff
@@ -50,24 +54,15 @@ class NetworkHost extends Model
...
-     * @var array
-     */
-    protected $traitAutomate = [
-        'address' => 'AccountAddress',
-    ];
-
     public static function registerHost($address, $host)
     {
-        $h = self::find(hex2bin(self::decodeAddress($address)));
+        $h = self::find($address);
```
Comment (teghnet) [inline app/Models/NetworkHost.php]: Add more meaningful name - i.e. `$host`

File: database/migrations/2018_04_06_103900_create_network_campaigns_table.php [hunk 2]
```diff
@@ -53,6 +53,9 @@ public function up()
             $table->decimal('budget_per_hour', 19, 11)->nullable(false);
             $table->dateTime('time_start');
             $table->dateTime('time_end');
+
+            $table->unsignedTinyInteger('status')->nullable(false)->default(1);
```
Comment (teghnet) [inline database/migrations/2018_04_06_103900_create_network_campaigns_table.php]: Please use `NetworCampaign::STATUS_x`

File: database/seeds/MockDataNetworkHostsSeeder.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with AdServer. If not, see <https://www.gnu.org/licenses/>
+ */
+
+use Adshares\Adserver\Models\NetworkHost;
+use Illuminate\Database\Seeder;
+
+class MockDataNetworkHostsSeeder extends Seeder
+{
+    /**
```
Comment (teghnet) [inline database/seeds/MockDataNetworkHostsSeeder.php]: Remove comment, maybe?

**Review by teghnet — CHANGES_REQUESTED** [review]

File: database/migrations/2018_04_06_103900_create_network_campaigns_table.php (hunk 2 above)
Comment (c3zi) [reply database/migrations/2018_04_06_103900_create_network_campaigns_table.php]: :+1:

File: app/Models/NetworkHost.php (hunk 1 above)
Comment (c3zi) [reply app/Models/NetworkHost.php]: :+1:

File: database/seeds/MockDataNetworkHostsSeeder.php (hunk 3 above)
Comment (c3zi) [reply database/seeds/MockDataNetworkHostsSeeder.php]: :+1:

**Review by teghnet — APPROVED** [review]

**Pull request merged**